v22.1/alter-default-privileges.md

```diff
@@ -32,7 +32,8 @@ Parameter | Description
 `FOR ALL ROLES` | Alter the default privileges on objects created by all users/roles.
 `GRANT ...` | Grant a default privilege or list of privileges on all objects of the specified type to a role/user, or a list of roles/users.
 `REVOKE ...` | Revoke a default privilege or list of privileges on all objects of the specified type from a role/user, or a list of roles/users.
-`IN SCHEMA qualifiable_schema_name` | This clause is a no-op. The syntax is supported for compatibility with PostgreSQL.
+`IN SCHEMA qualifiable_schema_name` | **New in v22.1:** If specified, the default privileges are altered for objects created in that schema. If an object has default privileges specified at
```

Comment:
```suggestion
`IN SCHEMA qualifiable_schema_name` | **{% include_cached new-in.html version="v22.1" %}** If specified, the default privileges are altered for objects created in that schema. If an object has default privileges specified at
```

Reply:
The Liquid tag doesn't work in tables.

Reply:
Ah I overlooked this. But it seems `<span class="version-tag">New in vXX.Y:</span>` is the guidance for tables.

Reply:
At one point we were using the format for version tags that's in this PR and it's also used in a number of other files. Since we have to do a global S&R for all those instances, I'd like to let this one go and fix them all at the same time. I'd also like to revisit the style for tables and plan to bring it up in the Style team. I think the `<span ...` is overkill.

Reply:
Works for me!